core/src/main/java/org/elasticsearch/index/query/CommonTermsQueryParser.java

```diff
@@ -73,6 +73,9 @@ public CommonTermsQueryBuilder fromXContent(QueryParseContext parseContext) thro
                                     throw new ParsingException(parser.getTokenLocation(), "[common] query does not support [" + innerFieldName
                                             + "] for [" + currentFieldName + "]");
                                 }
+                            } else {
+                                throw new ParsingException(parser.getTokenLocation(), "[common] unexpected token type [" + token
```

Comment:
You switched to using the query builder NAME in other places - why not here as well?